autoPyTorch/evaluation/tae.py

```diff
@@ -34,7 +34,7 @@
 
 def fit_predict_try_except_decorator(
         ta: typing.Callable,
-        queue: multiprocessing.Queue, cost_for_crash: float, **kwargs: typing.Any) -> None:
+        queue: multiprocessing.Queue, cost_for_crash: float, **kwargs: typing.Any) -> typing.Optional[typing.Any]:
```

Comment:
Because we do not use any return in the code (we take the results from pynisher), we do not need to return.
So I think the following is better.
```
try:
    ta(...)
except:
    ...
```

Reply:
True, I'll change this